manifests/0000_90_openshift-controller-manager-operator_04_operand-route-controller-servicemonitor.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: prometheus-k8s
+  namespace: openshift-route-controller-manager
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - services
+  - endpoints
+  - pods
+  verbs:
+  - get
+  - list
+  - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
```

Comment:
This role binding is already defined in bindata/v3.11.0/openshift-controller-manager/route-controller-servicemonitor-rolebinding.yaml in https://github.com/openshift/cluster-openshift-controller-manager-operator/pull/255/commits/d505b981d68a79a02a0038dba60e40f6647b9a5e#diff-6683cf67175abdad3fc07c757a5fe0e2c62b0c88f642989b1873bcd4f3f224a0